proto/substrait/algebra.proto

```diff
@@ -228,6 +228,71 @@ message AggregateRel {
   }
 }
 
+// The Window relational operate on a group of rows, referred to as a window, and calculate a return value for each row based on the group of rows.
+message WindowRel {
+  RelCommon common = 1;
+  Rel input = 2;
+  repeated WindowRelFunction window_functions = 3;
+  repeated Expression partition_expressions = 4;
+  repeated SortField sorts = 5;
+
+  substrait.extensions.AdvancedExtension advanced_extension = 10;
+
+  // A window rel function call.
+  message WindowRelFunction {
+    // Points to a function_anchor defined in this plan, which must refer
+    // to a window function in the associated YAML file. Required; 0 is
+    // considered to be a valid anchor/reference.
+    uint32 function_reference = 1;
+
+    // The arguments to be bound to the function. This must have exactly the
+    // number of arguments specified in the function definition, and the
```

Comment:
I can't come up with a good reason why aggregation phase was removed. The reason for a distinct expression-like thing is to move partition and sort out of the object. Let's not extra differences that will confuse people unnecessarily. (one of the reasons that this kind of duplication is something I'd prefer to work harder to avoid).

Reply:
It seems these properties are not used in Spark. And we can also remain here for the convenience of other systems if necessary.

Reply:
Please keep it.